rust/worker/src/memberlist/memberlist_provider.rs

```
@@ -168,7 +168,7 @@ impl CustomResourceMemberlistProvider {
     async fn notify_subscribers(&self) -> () {
         let curr_memberlist = match self.current_memberlist.read() {
             Ok(curr_memberlist) => curr_memberlist.clone(),
-            Err(err) => {
+            Err(_err) => {
```

I would prefer we didn't mask errors so the warnings tell us to clean this up.